src/lib/isJWT.js

```diff
@@ -7,7 +7,7 @@ export default function isJWT(str) {
   const dotSplit = str.split('.');
   const len = dotSplit.length;
 
-  if (len > 3 || len < 2) {
+  if (len !== 3) {
```

Comment:
consider adding reference link eg. https://jwt.io/introduction

Reply:
Thanks Chrislemus! for your valuable feedback.